ui/components/suite.jsx

```diff
@@ -1,8 +1,29 @@
-var React = require('react');
+var React = require('react'),
+    Test = require('./test.jsx');
 
 var Suite = React.createClass({
+  statics: {
+    MARGIN_LEFT: 20,
+    FONT_SIZE: 100
+  },
   render: function() {
-    return null;
+    var suite = this.props.suite,
+        suiteStyle = {
+          marginLeft: this.constructor.MARGIN_LEFT * (suite.indent - 1) + 'px'
```

Comment:
Look closely at the test: 'Expected `680` (without px) to equal `680px` (with px)'. You need to update the test and stop adding `px` there too :smile: